content/js/access-control.md

```diff
@@ -1 +1,119 @@
-Coming soon
+<a name='acl-type'></a>
+## Access Type
+
+There are two types of access for each record: read and write.
+- Read access allows querying and fetching records, which includes getting
+  all the fields of records as well as the access control settings.
+- Write access allows saving and deleting records, which includes adding,
+  updating and removing all fields of records as well as access control
+  settings for the record.
+
+However, even with write access, ownership of records cannot be changed.
+
+<a name='acl-default'></a>
+## Default ACL Settings
+
+By default, read access is granted to the public and write access is granted
+only to owner of the record. You can change this behavior as well:
```

Comment:
need to mention the fact that this behavior will affect all records created (how about updating an existing record?)